conf/rule-type-list.json

```
@@ -1,11 +1,8 @@
 {
-    "categories": [
-        { "name": "Possible Errors", "description": "These rules relate to possible syntax or logic errors in JavaScript code:" },
-        { "name": "Best Practices", "description": "These rules relate to better ways of doing things to help you avoid problems:" },
-        { "name": "Strict Mode", "description": "These rules relate to strict mode directives:" },
-        { "name": "Variables", "description": "These rules relate to variable declarations:" },
-        { "name": "Stylistic Issues", "description": "These rules relate to style guidelines, and are therefore quite subjective:" },
-        { "name": "ECMAScript 6", "description": "These rules relate to ES6, also known as ES2015:" }
+    "types": [
+        { "name": "problem", "displayName": "Possible Problems", "description": "Warns about a potential error." },
+        { "name": "suggestion", "displayName": "Suggestions", "description": "Suggests an alternate way of doing something." },
+        { "name": "layout", "displayName": "Layout & Formatting", "description": "Enforces code style." }
```

```suggestion
        { "name": "problem", "displayName": "Possible Problems", "description": "These rules relate to possible logic errors in code:" },
        { "name": "suggestion", "displayName": "Suggestions", "description": "These rules suggest alternate ways of doing things:" },
        { "name": "layout", "displayName": "Layout & Formatting", "description": "These rules care about how the code looks rather than how it executes:" }
```

Thoughts about something like this?

(related to https://github.com/eslint/website/pull/848#issuecomment-867571119)